README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
 
 ```python
-engine_string='sqlite:///data/tracks.db'
-
+engine_string='sqlite:///data/games.db'
 ```
 
-The three `///` denote that it is a relative path to where the code is being run (which is from the root of this directory).
+##### RDS Instance database:
 
-You can also define the absolute path with four `////`, for example:
+Make sure you are connected to northwestern vpn
 
-```python
-engine_string = 'sqlite://///Users/cmawer/Repos/2020-MSIA423-template-repository/data/tracks.db'
-```
+Go to step 3 and 4 and check environment variables and requirement.(check database name)
+
+ Go to the config directory and in `config.py` change local variable:
+    
+    local=False
+ 
+ Go to the root directory and create the rds instance database schema by running:
+       
+    python create_db.py
+ 
+ You may see a warning command here - invalid String in Game - there are games with special characters.
+ PyMySQL cannot decode them but sqlite3 can.
+ 
+ Your data is now in the database, with the schema defined, and the raw data has also been added to your S3 bucket! If you've used an RDS instance, you can verify that things have worked as expected by using the MySQL client. Run the `run_mysql_client.sh` script that allows connection to your SQL database:
+ 
+ In the root directory run (for mac users):
+ 
+    sh run_mysql_client.sh
+    
+ For windows in git bash run:
+ 	
+	winpty sh run_mysql_client.sh
+
+ You can query the customer, steam, metacritic and sales table in the `msia423asg5718` database as follows:
+    
+    use msia423asg5718;
+    
+Check if the 4 tables are there:
+    
+    show msia423asg5718;
+Output is as follows:
+    
+    +--------------------------+
+    | Tables_in_msia423asg5718 |
+    +--------------------------+
+    | customer                 |
+    | metacritic               |
+    | sales                    |
+    | steam                    |
+    +--------------------------+
+    4 rows in set (0.04 sec)
+
+Now run the following to check the steam table:
+    
+    SELECT * FROM steam LIMIT 5;
+ 
+The output of the query is the following, do validate:
+
+    +----+-----------+----------------------------+----------+
+    | id | Userid    | Game                       | Playtime |
+    +----+-----------+----------------------------+----------+
+    |  1 | 151603712 | The Elder Scrolls V Skyrim |      273 |
+    |  2 | 151603712 | Fallout 4                  |       87 |
+    |  3 | 151603712 | Spore                      |     14.9 |
+    |  4 | 151603712 | Fallout New Vegas          |     12.1 |
+    |  5 | 151603712 | Left 4 Dead 2              |      8.9 |
+    +----+-----------+----------------------------+----------+
+    5 rows in set (0.04 sec)
+
+Do validate this with the other tables as well!
+
+To exit mysql:
+    
+    exit;
+ 
+ To exit environment if in one:
+ 
```

Comment:
conda environments (and installing pip) proved uncesseary since we are not really running your create_db.py file (we don't have write access). the only thing being run is your push_s3.py file, and that is being run in a docker container.

Reply:
Noted. Will remove this.